app/src/main/java/me/ivg2/foodapp/RecipeItemRepository.java

```diff
@@ -12,8 +12,20 @@ private void load() {
         recipes = new ArrayList<>();
 
         //input manual data for recipes for reference
-        recipes.add(new Recipe("recipe1", "https://imagesvc.timeincapp.com/v3/mm/image?url=http%3A%2F%2Fcdn-image.myrecipes.com%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2Fimage%2Frecipes%2Fck%2F11%2F04%2Ffettuccine-olive-oil-ck-x.jpg%3Fitok%3DN9u99OOY&w=700&q=85", "mine", 1, 12));
-        recipes.add(new Recipe("recipe2", "https://imagesvc.timeincapp.com/v3/mm/image?url=https%3A%2F%2Fimg1.southernliving.timeinc.net%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2Fimage%2F2015%2F11%2Fmain%2Ffofoma051120100_0.jpg%3Fitok%3DuG-PCDzS&w=700&q=85", "mine", 0, 12));
+        recipes.add(new Recipe("Spaghetii Carbonara", "https://imagesvc.timeincapp.com/v3/mm/image?url=http%3A%2F%2Fcdn-image.myrecipes.com%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2Fimage%2Frecipes%2Fck%2F11%2F04%2Ffettuccine-olive-oil-ck-x.jpg%3Fitok%3DN9u99OOY&w=700&q=85", "Food.com", 1, 12));
+        recipes.add(new Recipe("Ravioli", "https://imagesvc.timeincapp.com/v3/mm/image?url=https%3A%2F%2Fimg1.southernliving.timeinc.net%2Fsites%2Fdefault%2Ffiles%2Fstyles%2Fmedium_2x%2Fpublic%2Fimage%2F2015%2F11%2Fmain%2Ffofoma051120100_0.jpg%3Fitok%3DuG-PCDzS&w=700&q=85", "Recipes.com", 0, 45));
+
+        ArrayList<String> instructions = new ArrayList<>();
+        ArrayList<String> ingredients = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
```

Comment:
Doesn't actually matter, but remember that you can do a loop from 1 to 11 instead of 0 through 10 with +1s :)